tools/RAiDER/models/hrrr.py

```diff
@@ -226,6 +232,25 @@ def __init__(self):
         self._proj = p1
 
         self._valid_bounds =  Polygon(((-125, 21), (-133, 49), (-60, 49), (-72, 21)))
+        self.setLevelType('ml')
```

Comment:
This we should be coupling to the "nat" variable as that could be changed and this could still be coded up to say model levels.

 So i would catch it here in a statement and if prs is provided it should go differently and could throw and error saying this does not reach high enough altitude or so?

Reply:
We don't allow for this in any of the other models as of now; i.e., we always force model levels.
I added a change to for coupling to 'nat' though to support it in the future if we want